### PR TITLE
Ayeletbinder/tabs component

### PR DESCRIPTION
**Pull request by ayeletBinder**
i did the tabs and the header components with styles(scss), in addition i install the scss in the project.
can you allow my code?
thank you.

### CODE REVIEW

**Review by CohenAvigail — APPROVED** [review]
looks good!

**Comment by ayeletBinder** [comment]
can you review again my code?
thanks!

**Review by ruth4453 — APPROVED** [review]

File: client-side/src/components/tabs.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    setValue(newValue);
+  };
+
+  const navigate = useNavigate();
+
+  const handleTabClick = (index) => {
+    if (nav && nav[index]) {
+      navigate(nav[index]);
+    }
+    debugger
```
Comment (e171271) [inline client-side/src/components/tabs.jsx:24]: please remove debugger

File: client-side/src/index.js [hunk 2]
```diff
@@ -3,11 +3,17 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
+import { createTheme } from '@mui/material';
+import { ThemeProvider } from '@emotion/react';
+
```
Comment (e171271) [inline client-side/src/index.js:9]: please re-order the imports

File: client-side/src/components/stylies/tabs.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap');
+$primary-color: black;
+$secondary-color: rgb(103, 252, 210);
+.div{
+  .every-tabs,
+  .profile-tabs,
+  .footer-tabs,
+  .navbar-tabs{
+      background-color: $primary-color   ;
+      color: $secondary-color !important ;
```
[the remark this answers is not in the document]
Comment (ayeletBinder) [reply client-side/src/components/stylies/tabs.scss:11]: עשיתי את זה רק פה כיוון שאין אפשרות לדרוס את ההגדרה הזו.

File: client-side/src/App.js [hunk 4]
```diff
@@ -1,10 +1,13 @@
 
-import './App.css';
+import { Route, Routes } from 'react-router';
+import './App.scss';
```
[the remark this answers is not in the document]
Comment (e171271) [reply client-side/src/App.js:5]: approved fix

File: client-side/src/components/stylies/tabs.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (e171271) [reply client-side/src/components/stylies/tabs.scss]: why you can't override this ?

File: client-side/src/components/tabs.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+import Box from '@mui/material/Box';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+import './stylies/tabs.scss'
+
+
+export default function LabTabs({ nameOfClass = "every-tabs", text = [], nav=[]}) {
+  const [value, setValue] = React.useState(0);
```
[the remark this answers is not in the document]
Comment (e171271) [reply client-side/src/components/tabs.jsx]: approved fix

**Review by e171271 — CHANGES_REQUESTED** [review]

**Review by e171271 — APPROVED** [review]

**Pull request merged**